dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.cpp

```diff
@@ -322,23 +322,23 @@ void VerifyInitializationIsCompatible(
     GetFrameworkPackageFamilyName(majorMinorVersion, versionTag);
 
     // Is the initialization request compatible with the current initialization state?
-    THROW_HR_IF_MSG(MDD_E_BOOTSTRAP_INITIALIZE_INCOMPATIBLE,
+    THROW_HR_IF_MSG(E_FAIL,
```

Comment:
Does this HRESULT find its way to user code?  If so, we can't change it.  (See Liskov Substitution Principle)

Reply:
it is inside a void functrion so I assume, no.
 still user can have a try catch block with this error. i will replace with original error message from Win32

Reply:
right - return type of the function is irrelevant to exceptions thrown